controllers/v1/cloud-services/aws.js

```diff
@@ -179,5 +180,97 @@ module.exports = class Aws {
     })
    }
 
+ /**
+     * @api {post} /kendra/api/v1/cloud-services/aws/uploadFile  
+     * Upload file to AWS
+     * @apiVersion 1.0.0
+     * @apiGroup Aws
+     * @apiHeader {String} X-authenticated-user-token Authenticity token
+     * @apiParamExample {fromData} Request:
+     * {
+     * "filePath" : "1230981723091723/sample.csv"
+     * "bucketName":"sl-dev-storage",
+     * "storage": "GC"
+     *  
+     * }
+     * @apiSampleRequest /kendra/api/v1/cloud-services/aws/uploadFile
+     * @apiUse successBody
+     * @apiUse errorBody
+     * @apiParamExample {json} Response:
+     * {
+     * "status": 200,
+     *  "message":"File uploaded successfully",
+     *  "result": {
+     *   "kind": "storage#object",
+     *   "id": "sl-dev-storage/my.csv/1590132715837085",
+     *   "selfLink": "https://www.googleapis.com/storage/v1/b/sl-dev-storage/o/my.csv",
+     *   "mediaLink": "https://storage.googleapis.com/download/storage/v1/b/sl-dev-storage/o/my.csv?generation=1590132715837085&alt=media",
+     *   "name": "my.csv",
+     *   "bucket": "sl-dev-storage",
+     *   "generation": "1590132715837085",
+     *   "metageneration": "1",
+     *   "contentType": "text/csv; charset=utf-8",
+     *   "storageClass": "REGIONAL",
+     *   "size": "137",
+     *   "md5Hash": "edPW+5pdDV6cbSl1TkVwLA==",
+     *   "contentEncoding": "gzip",
+     *   "crc32c": "56+5+A==",
+     *   "etag": "CJ2tsf35xukCEAE=",
+     *   "timeCreated": "2020-05-22T07:31:55.836Z",
+     *   "updated": "2020-05-22T07:31:55.836Z",
+     *   "timeStorageClassUpdated": "2020-05-22T07:31:55.836Z"
+     *    }
+     * }
+     */
+
+    /**
+      * Upload file to AWS
+      * @method
+      * @name uploadFile
+      * @param  {Request}  req  request body.
+      * @param  {files}  req.files.file -actual file to upload
+      * @param  {String}  req.body.bucketName - bucket name
+      * @param  {String}  req.body.filePath - file path of where to store
+      * @returns {JSON} Response with status and message.
+    */
+   async uploadFile(req) {
+    return new Promise(async (resolve, reject) => {
+
+        try {
+            if (req.files && req.body.filePath && req.body.bucketName){
```

Comment:
@rakeshSgr - Add a space before the curly brace

Reply:
resolved